packages/commonwealth/client/scripts/navigation/CommonDomainRoutes.tsx

```diff
@@ -115,7 +116,17 @@ const CommonDomainRoutes = ({
   farcasterContestEnabled,
   tokenizedCommunityEnabled,
 }: RouteFeatureFlags) => [
-  <Route key="/editor" path="/editor" element={<EditorPage />} />,
+  <Route
+    key="/markdown-editor"
+    path="/markdown-editor"
+    element={<MarkdownEditorPage />}
+  />,
+
+  <Route
+    key="/markdown-viewer"
+    path="/markdown-viewer"
+    element={<MarkdownViewerPage />}
+  />,
```

Comment:
lets add these behind `FLAG_NEW_EDITOR`

Reply:
They're just for testing.  We could put them under a prefix like /_/markdown-editor...

They don't need to be behind a flag or hidden as no one will actually use them.. They're not linked anywhere.